README.md

```diff
@@ -73,75 +73,13 @@ For more information refer to the [architecture](./docs/ARCHITECTURE.md). Note t
 1. Run the demo
 
     ```shell
-    # Start the Docker containers
```

Comment:
optional but I think it's nice to demonstrate what steps go into `make demo`

Reply:
sg, https://github.com/celestiaorg/celestia-zkevm-ibc-demo/pull/226/commits/3a24f68534fd3d84fa61947999a73bcb8fce668e